<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="passwordfield" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.0678571, y1:0.102, x2:1, y2:1, stop:0 rgba(0, 230, 255, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Sign in to your existing account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_edit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:rgba(0,0,0,0);
	border: 2px solid white;
	border-radius: 5px;
	padding: 5px;
}

QLineEdit::focus{
	border: 2px solid blue;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:rgba(0,0,0,0);
	border: 2px solid white;
	border-radius: 5px;
	padding: 5px;
}

QLineEdit::focus{
	border: 2px solid blue;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>145</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QWidget" name="bgwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>990</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.542289, y1:0.057, x2:0.56204, y2:0.903, stop:0 rgba(91, 139, 223, 255), stop:1 rgba(6, 201, 229, 255));}</string>
    </property>
    <widget class="QLabel" name="error_field">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>290</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:20px;
	background-color: rgb(45, 101, 197);
	font: 13pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="redirect_login">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;New to Taskify? &lt;a href=&quot;signup_page&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Sign up.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_logo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="login_resource.qrc">:/newPrefix/assets/login_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:20px;
	background-color: rgb(45, 101, 197);
	font: 13pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0,0,0);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <zorder>bgwidget</zorder>
   <zorder>label_2</zorder>
   <zorder>username_edit</zorder>
   <zorder>password_edit</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="login_resource.qrc"/>
 </resources>
 <connections/>
</ui>
